### ISSUE TITLE
support custom installation hints

### ISSUE THREAD

**Issue by jayunit100** [Masterminds/glide#680]
For projects like kubernetes which generate data as part of compilation, how should we compile those projects when pulling them in as glide dependencies ?

For example is `go:generate` always/sometimes honored ?

**Comment by sdboyer** [comment]
I don't think we have a good story for this right now, sadly. It's something I've been loathe to delve too far into until we've solved more fundamental problems with version selection and checking out code, as directly including support for this sort of thing as part of the tool amounts to arbitrary code execution, and thus is a significant attack vector.

So, yeah, for now, glide does none of those sorts of things. Standard practice would be to wrap up the necessary commands in your build tool, which is (presumably) also invoking glide.
